woocommerce/woocommerce-gutenberg-products-block
# Add previews to some Product grid blocks

Fixes #1030.

This PR adds previews to: _Products on Sale_, _Best Selling Products_, _Newest Products_ and _Top Rated Products_ blocks and make it easy to add previews to other product grid blocks.

### Screenshots

![Peek 2019-10-23 14-41](https://user-images.githubusercontent.com/3616980/67393717-376ee680-f5a3-11e9-9d0b-0f26a87c70e7.gif)

### How to test the changes in this Pull Request:

1. Create a post and, using the top bar inserter, hover the _Products on Sale_, _Best Selling Products_, _Newest Products_ and _Top Rated Products_ blocks.
2. Verify all of them show a preview of the block.

### Changelog

> Added Previews to  Products on Sale, Best Selling Products, Newest Products and Top Rated Products blocks.
